src/Generator.Rewrite/AssemblyResolver.cs

```diff
@@ -2,7 +2,12 @@
 
 namespace OpenTK.Rewrite
 {
+    /// <summary>
+    /// A class that is responsible for resolving assembly references.<para/>
```

Comment:
\<para/> has no opening

Reply:
It's `<para/>`, not `</para>`. This is a self closing XML element and is thus both opening and closing tag in one

Reply:
My bad)